### ISSUE TITLE
ACCESSIBILITY_TWEAKS=true results in invalid form labels in contact form

### ISSUE THREAD

**Issue by JuliaGru** [shopware/platform#5439]
*PHP Version:* 8.3   | *Shopware Version:* 6.6.8.1  | *Affected area / extension:* Platform(Default) 

 ----

*_Expected behaviour:_*

The IDs of the input fields should be referenced in the labels with for="".

*_Actual behaviour:_*

Is not the same, since the formPrefix is not used in form label:

`for="form-{{ fieldName }}"`
`id="{% if feature('ACCESSIBILITY_TWEAKS') %}{{ formPrefix }}-{% endif %}form-{{ fieldName }}"`

*_How to reproduce:_*

Inspect contact form or have a look in a template fg. `/element/cms-element-form/form-components/cms-element-form-input.html.twig`

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-39498**



[created from NEXT-39498, comment 511398]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.9.0**

 This issue will be / was solved in Shopware version ***6.6.9.0***.



[created from NEXT-39498, comment 526946]
